helm/superset/values.yaml

```diff
@@ -54,11 +54,21 @@ extraEnv: {}
   # GUNICORN_TIMEOUT: 300
 
 
-   # OAUTH_HOME_DOMAIN: ..
+  # OAUTH_HOME_DOMAIN: ..
   # # If a whitelist is not set, any address that can use your OAuth2 endpoint will be able to login.
   # #   this includes any random Gmail address if your OAuth2 Web App is set to External.
   # OAUTH_WHITELIST_REGEX: ...
 
+## Extra environment variables in RAW format that will be passed into pods
+##
+extraEnvRaw: {}
```

Comment:
I think the default should be set to empty here. Using `{}` as the default is a little misleading as it's a dictionary type whereas we need a list instead.

Reply:
fixed